onadata/libs/serializers/xform_serializer.py

```diff
@@ -328,6 +328,28 @@ def validate_public_key(self, value):  # pylint: disable=no-self-use
                 _('The public key is not a valid base64 RSA key'))
         return clean_public_key(value)
 
+    def _check_if_allowed_public(self, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5585241)